src/main/java/be/syntra/devshop/DevshopBack/services/DataFillerServiceImpl.java

```diff
@@ -180,6 +185,35 @@ public void initialize() {
                             "admin@emaill.com")
             ));
         }
+
+        if (cartRepository.count() == 0) {
+            userRepository.findAll().forEach(this::setArchivedCartsAndSave);
+            userRepository.flush();
+        }
+    }
+
+    private void setArchivedCartsAndSave(User user) {
+        user.setArchivedShopOrders(List.of(createCart()));
+    }
+
+    private ShopOrder createCart() {
+        return shopOrderFactory.of(getListOfRandomProducts(), true, true, getCartContent());
+    }
+
+    private List<Product> getListOfRandomProducts() {
+        List<Product> products = new ArrayList<>();
```

Comment:
oneliner equivalent: return IntStream.range(0,6).boxed().map(number -> productRepository.findAll().stream().findAny().orElseThrow()).collect(Collectors.toList());

Reply:
after fixes above this whole method wasn't needed anymore, it seems that i did the choosing of random products elsewhere at a later stage and forgot to remove it here